git-keeper-client/gkeepclient/client_configuration.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2016 Nathan Sommer and Ben Coleman
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import configparser
+import os
+
+
+# Store one module-level ClientConfiguration instance
+config_instance = None
+
+
+class ConfigurationError(Exception):
+    pass
+
+
+class ClientConfiguration:
+    """Parses client.cfg and stores the configuration values as attributes.
```

Comment:
A link to https://docs.python.org/3/library/configparser.html#supported-ini-file-structure to describe the expected file format would be helpful.